### DISCUSSION TITLE
Vision capabilitie for OpenAI assistant

### DISCUSSION THREAD

**Discussion by robertsmaoui** [danny-avila/LibreChat#3897] [Troubleshooting]
### What happened?

OpenAI assistant not detect image content when sending image in chat 

but it works fine with openai model like gpt4o 

### Steps to Reproduce

in librechat.yaml :

endpoints:
   assistants:
     disableBuilder: false # Disable Assistants Builder Interface by setting to `tru
     ASSISTANTS_API_KEY: "sk-proj-rPKccccccccccTsA"
     capabilities:
        - "tools"
        - "image_vision"
 and .env : 

ENDPOINTS=openAI,assistants

OPENAI_API_KEY=sk-proj-rPhhhhhhhhhhhhhhhhhhhhhhhsA
OPENAI_MODELS=gpt-4o-mini

ASSISTANTS_API_KEY=sk-proj-rPK3XvvvvvvvvvvvvvvvvvvvvvvvvvNKlTsA
ASSISTANTS_MODELS=gpt-4o-mini

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
The “image vision” capability is no longer used since OpenAI started supporting vision natively with supported models.

use a supported vision model, gpt-4o is one of them

**Reply by robertsmaoui** [reply]
The vision not works in librechat in openai assistant chat !!
![Capture d’écran 2024-09-02 141304](https://github.com/user-attachments/assets/aeaf502c-e3d9-45a6-88a9-96f3f70714aa)

**Reply by danny-avila** [reply]
I investigated it and it looks like OpenAI changed the way they are handling images. Will look into it.

**Reply by robertsmaoui** [reply]
Thank you !

**Reply by faniereynders** [reply]
any update on this?
